shopfloor/services/checkout.py

```diff
@@ -1405,13 +1405,14 @@ def done(self, picking_id, confirmation=False):
                 )
         lines_done = self._lines_checkout_done(picking)
         dest_location = picking.location_dest_id
```

Comment:
Should be the move destination location

Reply:
There is something fishy there:

1. as said, we should take the destination location of the move
2. but what move? we get all lines done of `picking`, so we could get several moves as well, each having a different destination location (e.g. a sub-location of the picking destination location thanks to a put-away)
3. so we should ask the user to scan a relevant leaf location for each move that targets a non-leaf location?

Right now this code is asking the user to scan a leaf location even if it's probably not necessary IMO :thinking:

Reply:
We should always scan a destination if the moves target different destinations